proposals/0282-screen-manager-alerts.md

```diff
@@ -245,7 +240,18 @@ And then the additions to the screen manager public API itself to present the al
 ##### Java
 ```java
 public class BaseScreenManager {
-    public void presentAlert(AlertView alert, CompletionListener listener)
+    // Everything already there
+
+    public void presentAlert(AlertView alert, AlertCompletionListener listener)
+}
+
+public interface AlertCompletionListener {
+    /**
+     * Returns whether an Alert operation was successful or not along with tryAgainTime
+     * @param success - Boolean that is True if Operation was a success, False otherwise.
+     * @param tryAgainTime - Amount of time (in seconds) that an app must wait before resending an alert.
+     */
+    void onComplete(boolean success, Integer tryAgainTime);
```

Comment:
Should we separate this into two blocks of code, I don't want to give the impression that the AlertCompletionListener is in the same file as BaseScreenManager, as it is its own file

Reply:
Separated into two blocks